Library/Homebrew/sorbet/rbi/dsl/.gitattributes

```diff
@@ -0,0 +1 @@
+**/*.rbi linguist-generated=true
```

Comment:
This was put here by tapioca, and is new to me: https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github

I think I don't like it, at least for our use case (the largest file in here is 34 lines), but let's keep the default behavior for now.

Reply:
I agree it doesn't make sense for our case, but there doesn't seem to be a way to disable it currently.

Reply:
(specifically, i was thinking we'd remove this file after the call to `tapioca dsl` here: https://github.com/Homebrew/brew/pull/16586/files#diff-e2fa03718f64a9a32fe9acef1ced320316bbb31161cbbaa74077c88cf24819feR48 )